milli/src/lib.rs

```diff
@@ -143,7 +143,7 @@ where
 }
 
 /// Return the distance between two points in meters.
-fn distance_between_two_points(a: &[f64; 2], b: &[f64; 2]) -> f64 {
+pub fn distance_between_two_points(a: &[f64; 2], b: &[f64; 2]) -> f64 {
```

Comment:
Can we add a description that specifies that the points are long/lat points, please?

Reply:
done